sdk/python/v1beta1/kubeflow/katib/api/katib_client_test.py

```diff
@@ -0,0 +1,55 @@
+import multiprocessing
+import pytest
+from unittest.mock import patch, Mock
+from kubeflow.katib import models
+from kubeflow.katib.api.katib_client import KatibClient
+
+@pytest.fixture
+def katib_client():
+    return KatibClient()
+
+@pytest.fixture
+def valid_experiment():
+    return models.V1beta1Experiment(
+        metadata=models.V1ObjectMeta(name="test-experiment"),
+        spec=models.V1beta1ExperimentSpec()
+    )
+
+class TestCreateExperiment:
+    @patch('kubeflow.katib.api.katib_client.client.CustomObjectsApi.create_namespaced_custom_object')
+    def test_create_experiment_success(self, mock_create_namespaced_custom_object, katib_client, valid_experiment):
```

Comment:
I think, it would be nice if you could add unit test to verify these conditions: https://github.com/kubeflow/katib/blob/master/sdk/python/v1beta1/kubeflow/katib/api/katib_client.py#L96-L108.
E.g. you can create create `test_data` list under TestCreateExperiment class and add various test cases there, similar to [TrainingClient tests](https://github.com/kubeflow/training-operator/blob/master/sdk/python/kubeflow/training/api/training_client_test.py#L90).